package lesson10;

public class Demo01 {

	public static void main(String[] args) {
		//throw的概述以及和throws的区别
	/*	1.throw的使用场景
		在方法内部出现某种情况，程序不能继续运行，就用throw把异常对象抛出。
			
		2.throws和throw的区别
		throws
		用在方法声明后面，跟的是异常类名
		可以跟多个异常类名，用逗号隔开
		它表示抛出异常，由该方法的调用者来处理
		throw
		用在方法体内，跟的是异常对象名
		只能抛出一个异常对象名，表示抛出异常*/

	}
}
